services/gundeck/gundeck.integration.yaml

```diff
@@ -24,6 +24,9 @@ settings:
   httpPoolSize: 1024
   notificationTTL: 24192200
   bulkPush: true
+  nativePushBudget:  # 20 push notifications every 20 seconds.
+    budgetTimeout: 20
```

Comment:
Can we come up with a more scientific number? Can you add a comment that is a bit more descriptive? Is this the push budget per user? Or for the entire server? For the entire sever it seems a bit low no?

Reply:
the new approach (if you look at the overall changes now) solves this: it keeps track of the number of `push1` calls that are currently still being processed in some other thread, and makes sure that number does not exceed a certain threshold (well, at least heuristically, which should be good enough).